packages/bruno-app/src/components/Sidebar/NewRequest/index.js

```diff
@@ -12,6 +12,8 @@ import HttpMethodSelector from 'components/RequestPane/QueryUrl/HttpMethodSelect
 import { getDefaultRequestPaneTab } from 'utils/collections';
 import StyledWrapper from './StyledWrapper';
 import { getRequestFromCurlCommand } from 'utils/curl';
+import { variableNameRegex } from 'utils/common/regex';
+import { isWindowsOS } from 'utils/common/platform';
```

Comment:
I think it's a mistake to have separate rules for validation depending on the plafrom. The same collection should be usable on all supported platforms, or else the collaboration becomes a hassle.

Reply:
Hey, thanks for reviewing.
I agree with you, I just changed the code to make the validation across all platforms, is that okay for you? :)